pkg/apis/operators/v1/zz_generated.defaults.go

```diff
@@ -0,0 +1,32 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by defaulter-gen. DO NOT EDIT.
+
+package v1
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
+)
+
+// RegisterDefaults adds defaulters functions to the given scheme.
+// Public to allow building arbitrary schemes.
+// All generated defaulters are covering - they call all nested defaulters.
+func RegisterDefaults(scheme *runtime.Scheme) error {
+	return nil
+}
```

Comment:
I assume this file was generated by the latest version of the sdk?

Reply:
That is correct @kevinrizza.

This file was generated with version `v0.8.1` of the operator-sdk when I updated the CSC CRD and ran `$ operator-sdk generate k8s`.

Reply:
We are using `v0.8.0` in our code base. We should stick to using that SDK version also when generating code.

Reply:
Be careful here with what is installed and run locally. We are on `v0.8.0`: https://github.com/operator-framework/operator-marketplace/blob/master/Gopkg.toml#L70